web/src/narrow.js

```diff
@@ -1100,13 +1100,13 @@ export function deactivate() {
         narrow_state.set_has_shown_message_list_view();
 
         message_lists.update_current_message_list(message_lists.home);
-        message_lists.current.resume_reading();
+        message_lists.current?.resume_reading();
```

Comment:
This can be better done as an assert just after `message_lists.update_current_message_list(message_lists.home);`, since we know that it's set to `home` after that call.

Reply:
Yeah, changed.